packages/amplify_api_rest/android/src/main/kotlin/com/amazonaws/amplify/amplify_api_rest/types/inputs/FlutterRestInputs.kt

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
In all places in this PR where it is referenced:
```suggestion
 * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Reply:
ah good call ... will update 
and now soon to 2021!